### ISSUE TITLE
The state of the power toggle buttons in realtime effects dialogs are not accessible for users of screen readers

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#3482]
### Bug description

The state of the power toggle buttons in realtime effects dialogs are not accessible for users of any of the screen readers on Windows.

### Steps to reproduce

1. Open a realtime effects dialog.
2.  Tab to the power toggle button.
3.  Press Enter to press the power toggle button.
4.  Tab away from the button and then back to it to read it in its changed state.

### Expected behavior

When a toggle button becomes the focus, the state of the button should always be read. Screen readers on Windows differ in the exact words used, but NVDA reads either pressed or not pressed.

When a toggle button is pressed, the screen reader reads the new state, for example pressed or not pressed.

### Actual behavior

When the power toggle button becomes the focus, all screen readers just read the name of the button - "Enable button"

When the power toggle button is pressed, all of the screen readers are silent.

### Audacity Version

current master / alpha / nightly build

### Operating system

Windows

### Additional context

The power toggle button replaces an enable check box in previous versions of Audacity. The state of the check box was accessible for users of screen readers, and so this bug is a regression.

See related issue: https://github.com/audacity/audacity/issues/3470

**Comment by DavidBailes** [comment]
@LWinterberg . Since I've submitted a pull request to fix this, can this issue be added to the 3.3 milestone?

**Comment by Ryan-Miller-Audacity** [comment]
@DavidBailes I've tested this on Windows and confirmed that this is now working as the expected behavior describes above; however, my only concern being, the "Master" power button and the "individual" effect power buttons are labeled/read the exact same. I believe this could be confusing to Visually Impaired users. 

Should we change the "Master" effect power button (the power button that controls status for the entire track's effect stack) to read as "Master power button" or something of this sort?

**Comment by DavidBailes** [comment]
> Should we change the "Master" effect power button (the power button that controls status for the entire track's effect stack) to read as "Master effect power button" or something of this sort?

I think "master" might be misunderstood to be something connected with a master bus (although there isn't one yet). An alternative might be "Track effects power button", or maybe just "Track power button", as it's in the realtime effects pane.

**Comment by Ryan-Miller-Audacity** [comment]
I think "Track effects power button" would make sense. I'll move the related tickets for this pull request forward and we can create an alternate ticket for renaming that. As long as @LWinterberg thinks that naming makes sense?

**Comment by Paul-Licameli** [comment]
Will require a re-test to merge #4334

**Comment by Ryan-Miller-Audacity** [comment]
Retested with PR #4334.
